pkg/github/pullrequests.go

```diff
@@ -947,199 +1066,205 @@ func CreatePullRequestReview(getClient GetClientFn, t translations.TranslationHe
+			// Now we have the GQL ID, we can create a review
+			var addPullRequestReviewMutation struct {
+				AddPullRequestReview struct {
+					PullRequestReview struct {
+						ID githubv4.ID // We don't need this, but a selector is required or GQL complains.
 					}
+				} `graphql:"addPullRequestReview(input: $input)"`
+			}
 
-					_, hasPosition := commentMap["position"].(float64)
-					_, hasLine := commentMap["line"].(float64)
-					_, hasSide := commentMap["side"].(string)
-					_, hasStartLine := commentMap["start_line"].(float64)
-					_, hasStartSide := commentMap["start_side"].(string)
-
-					switch {
-					case !hasPosition && !hasLine:
-						return mcp.NewToolResultError("each comment must have either position or line"), nil
-					case hasPosition && (hasLine || hasSide || hasStartLine || hasStartSide):
-						return mcp.NewToolResultError("position cannot be combined with line, side, start_line, or start_side"), nil
-					case hasStartSide && !hasSide:
-						return mcp.NewToolResultError("if start_side is provided, side must also be provided"), nil
-					}
+			if err := client.Mutate(
+				ctx,
+				&addPullRequestReviewMutation,
+				githubv4.AddPullRequestReviewInput{
+					PullRequestID: getPullRequestQuery.Repository.PullRequest.ID,
+					Body:          newGQLStringlike[githubv4.String](body),
+					Event:         newGQLStringlike[githubv4.PullRequestReviewEvent](event),
+					CommitOID:     newGQLStringlike[githubv4.GitObjectID](commitID),
+				},
+				nil,
+			); err != nil {
+				return mcp.NewToolResultError(err.Error()), nil
+			}
 
-					comment := &github.DraftReviewComment{
-						Path: github.Ptr(path),
-						Body: github.Ptr(body),
-					}
+			// Return nothing, just indicate success for the time being.
+			// In future, we may want to return the review ID, but for the moment, we're not leaking
+			// API implementation details to the LLM.
+			return mcp.NewToolResultText(""), nil
+		}
+}
 
-					if positionFloat, ok := commentMap["position"].(float64); ok {
-						comment.Position = github.Ptr(int(positionFloat))
-					} else if lineFloat, ok := commentMap["line"].(float64); ok {
-						comment.Line = github.Ptr(int(lineFloat))
-					}
-					if side, ok := commentMap["side"].(string); ok {
-						comment.Side = github.Ptr(side)
-					}
-					if startLineFloat, ok := commentMap["start_line"].(float64); ok {
-						comment.StartLine = github.Ptr(int(startLineFloat))
-					}
-					if startSide, ok := commentMap["start_side"].(string); ok {
-						comment.StartSide = github.Ptr(startSide)
-					}
+// CreatePendingPullRequestReview creates a tool to create a pending review on a pull request.
+func CreatePendingPullRequestReview(getGQLClient GetGQLClientFn, t translations.TranslationHelperFunc) (mcp.Tool, server.ToolHandlerFunc) {
+	return mcp.NewTool("create_pending_pull_request_review",
+			mcp.WithDescription(t("TOOL_CREATE_PENDING_PULL_REQUEST_REVIEW_DESCRIPTION", "Create a pending review for a pull request.")),
```

Comment:
```suggestion
			mcp.WithDescription(t("TOOL_CREATE_PENDING_PULL_REQUEST_REVIEW_DESCRIPTION", "Create a pending review for a pull request. Call this first before attempting to add comments to a pending review, and ultimately submitting it. A pending pull request review means a pull request review, it is pending because you create it first and submit it later, and the PR author will not see it until it is submitted.")),
```